<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topo_blend_widget</class>
 <widget class="QWidget" name="topo_blend_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Topo Blend options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Minimumal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMax"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Check option</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>TopoBlend tools</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="0">
       <widget class="QLabel" name="resLabel">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="numStepsLabel">
        <property name="text">
         <string>Num. steps</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>Experimental...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="linksButton">
        <property name="text">
         <string>Links</string>
        </property>
        <property name="icon">
         <iconset resource="topo-blend.qrc">
          <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save model...</string>
        </property>
        <property name="icon">
         <iconset resource="topo-blend.qrc">
          <normaloff>:/images/saveGraph.png</normaloff>:/images/saveGraph.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="genSimpleButton">
        <property name="text">
         <string>Gen 2 simple</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="genChairsButton">
        <property name="text">
         <string>Generate chairs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load model...</string>
        </property>
        <property name="icon">
         <iconset resource="topo-blend.qrc">
          <normaloff>:/images/openGraph.png</normaloff>:/images/openGraph.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset resource="topo-blend.qrc">
          <normaloff>:/images/cross-white.png</normaloff>:/images/cross-white.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="animationButton">
        <property name="text">
         <string>Animation</string>
        </property>
        <property name="icon">
         <iconset resource="topo-blend.qrc">
          <normaloff>:/images/animation.png</normaloff>:/images/animation.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="numSteps">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="voxelSize">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1024.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QPushButton" name="groupButton">
          <property name="text">
           <string>Group</string>
          </property>
          <property name="icon">
           <iconset resource="topo-blend.qrc">
            <normaloff>:/images/group.png</normaloff>:/images/group.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="blendButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Blend!</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="alignButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string>Align</string>
          </property>
          <property name="icon">
           <iconset resource="topo-blend.qrc">
            <normaloff>:/images/align.png</normaloff>:/images/align.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPushButton" name="computeCorrButton">
          <property name="text">
           <string>Correspondences</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Synthesis data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QPushButton" name="saveSynthButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="loadSynthButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="genSynthButton">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="outputCloudButton">
        <property name="text">
         <string>Point cloud..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="topo-blend.qrc"/>
 </resources>
 <connections/>
</ui>
